src/test/java/com/parasoft/report/transformer/XMLToSarifTest.java

```diff
@@ -306,4 +280,37 @@ private void testWithMockedLogger(Consumer<MockedStatic<Logger>> function) {
             function.accept(mockedLogger);
         }
     }
+
+    private void testXMLToSarif(String xmlFileName, String outputSarifFileName, String expectedSarifFileName, String projectRootPaths, String expectedProjectRootPaths) throws IOException {
+        File outputSarifFile = new File(TEST_RESOURCES_LOC, outputSarifFileName != null ? outputSarifFileName : expectedSarifFileName);
```

Comment:
Theoretically, when `outputSarifFileName == null`, we should use name of `xmlFileName` and add `.sarif` after it to be rigorous.